bin/inasafe.py

```diff
@@ -15,28 +15,36 @@
         cli: command line interface
     .. versionadded:: 3.2
 """
-import shutil
 import logging
 import os
-import tempfile
 
-from PyQt4.QtCore import QSettings
-from docopt import docopt, DocoptExit
 from qgis.core import (
     QgsRasterLayer,
     QgsVectorLayer,
     QgsRectangle,
     QgsMapLayerRegistry,
     QgsCoordinateReferenceSystem)
+from PyQt4.QtCore import QSettings
 
+from docopt import docopt, DocoptExit
 
 from safe.test.utilities import get_qgis_app
 # make sure this line executes first
 QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
 
+from safe.common.version import get_version
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->